tests/test_models.py

```diff
@@ -89,3 +94,110 @@ def test_string_conversion(self, method):
             user_id=user_id
         )
         self.assertEqual(method(customer_user), expected_to_str)
+
+
+@mark.django_db
+@ddt.ddt
+class TestEnterpriseCustomerBrandingConfiguration(unittest.TestCase):
+    """
+    Tests of the EnterpriseCustomerBrandingConfiguration model.
+    """
+
+    def get_enterprise_customer(self):
+        """
+        @:return EnterpriseCustomer object.
+        """
+        faker = FakerFactory.create()
+        customer_uuid = faker.uuid4()
+        return EnterpriseCustomerFactory(uuid=customer_uuid, name="QWERTY")
+
+    def test_logo_path(self):
+        """
+        Test path of image file should be enterprise/branding/<model.id>/<model_id>_logo.<ext>.lower().
+        """
+        file_mock = mock.MagicMock(spec=File, name='FileMock')
+        file_mock.name = 'test1.png'
+        file_mock.size = 240 * 1024
+        branding_config = EnterpriseCustomerBrandingConfiguration(
+            id=1,
+            enterprise_customer=self.get_enterprise_customer(),
+            logo=file_mock
+        )
+
+        storage_mock = mock.MagicMock(spec=Storage, name='StorageMock')
+        with mock.patch('django.core.files.storage.default_storage._wrapped', storage_mock):
+            path = logo_path(branding_config, branding_config.logo.name)
+            self.assertEqual(path, 'enterprise/branding/1/1_logo.png')
+
+    def test_branding_configuration_saving_successfully(self):
+        """
+        Test enterprise customer branding configuration saving successfully.
+        """
+        branding_config = EnterpriseCustomerBrandingConfiguration(
+            enterprise_customer=self.get_enterprise_customer(),
+            logo='test1.png'
+        )
+
+        storage_mock = mock.MagicMock(spec=Storage, name='StorageMock')
+        with mock.patch('django.core.files.storage.default_storage._wrapped', storage_mock):
+            branding_config.save()
+            self.assertEqual(EnterpriseCustomerBrandingConfiguration.objects.count(), 1)
+
+    @ddt.data(
+        (False, 350 * 1024),
+        (False, 251 * 1024),
+        (False, 250 * 1024),
+        (True, 2 * 1024),
+        (True, 249 * 1024),
+    )
+    @ddt.unpack
+    def test_image_size(self, valid_image, image_size):
+        """
+        Test image size, image_size < (250 * 1024) e.g. 250kb. See apps.py.
+        """
+        raised = False
+
+        file_mock = mock.MagicMock(spec=File, name='FileMock')
+        file_mock.name = 'test1.png'
+        file_mock.size = image_size
+        branding_configuration = EnterpriseCustomerBrandingConfiguration(
+            enterprise_customer=self.get_enterprise_customer(),
+            logo=file_mock
+        )
+        try:
+            branding_configuration.full_clean()
+        except ValidationError:
+            raised = True
```

Comment:
@asadiqbal08 nit: this block might be better written as something like

    if not valid_image:
        with self.assertRaises(ValidationError):
            branding_configuration.full_clean()
    else:
            branding_configuration.full_clean()  # exception here will fail the test

But this is a question of personal taste.